pages/index.html

```diff
@@ -59,44 +60,43 @@
     <div style="float: right; margin-top: 16px; padding-right: 8px;">
         <img src="{% static '/static/images/squiggle.png' %}">
     </div>-->
-  <p style="text-align: center;">The most advanced free DJ software can be yours in seconds.</p>
+  <p style="text-align: center;">DJ your way. For free.</p>
   <div style="clear: both;"></div>
 </div> <!-- gapfiller -->
 
 <div class="content">
-  <H1>{% trans "Incredible Features, Unbeatable Value" %}</H1>
+  <h1>{% trans "Flexible Features For All DJs" %}</h1>
```

Comment:
I don't mean to imply that every feature is for everyone; I mean to say that "Mixxx already covers many use-cases and is open to add your own" and "Mixxx can be tweaked to your needs". I think the wording "Flexible Features For All DJs" conveys this, but I'll try to come up with some alternatives.

I agree that we should try to counter the stigma against free things implying that they are poor quality. I think the wording proposed does this pretty well. The very first line mentions how Mixxx is free, then this line talks about how Mixxx is quality.